### ISSUE TITLE
Cypress 10 + cypress-tags 1.0.0 looking for cypress.json

### ISSUE THREAD

**Issue by pdisselhorst** [annaet/cypress-tags#220]
Currently on Cupress 10.3.1 and trying to add cypress-tags (version 1.0.0 showing in package-lock.json).

Whenever I attempt to run any cypress-tags commands, I see the following error:

Failed to read cypress.json, using default configuration
Spec files:  cypress/integration/**/*.feature
No matching tags found

Any ideas why it is still searching for cypress.json instead of the new cypress-config.ts for Cypress 10+? Is there a way I can override which config file it attempts to use?

**Comment by annaet** [comment]
Hi @pdisselhorst 

Are you sure that's a cypress-tags issue? Cypress is the one that will look up your cypress.json or cypress.config.ts file. Cypress-tags is just a pre-processor.

I would also suggest upgrading to `cypress-tags@1.1.0`  as there were a couple of bug fixes added.

**Comment by annaet** [comment]
@nikotomad can you share your `cypress.config.ts`?

**Comment by nikotomad** [comment]
> @nikotomad can you share your `cypress.config.ts`?

Yes, surely. I've hidden some values but the context is still there (keep in mind this is a custom config file for mobile but same happens with the main config entry point regardless)

![image](https://user-images.githubusercontent.com/10640643/191753919-28d8a356-b88b-47c7-9ab9-e3444f901008.png)

Appreciate you reaching back to us

**Comment by annaet** [comment]
Hi @nikotomad 

It looks like you're not actually using `cypress-tags` as your preprocessor. I don't think the plugin would be compatible cucumber for this reason, as they both filter tests in different ways.

As per this issue (https://github.com/infosum/cypress-tags/issues/29) we currently can't support running alongside other preprocessors.

I would suggest if you're using cucumber then you will need to use their inbuilt test filtering.
